queries.sql

```diff
@@ -40,3 +40,25 @@ SELECT COUNT (*) FROM animals WHERE neutered = false;
 SELECT AVG(wieght_kg) FROM animals;
 SELECT MAX(escape_attempts) FROM animals;
 SELECT species, MAX(wieght_kg) FROM animals GROUP BY species;
+
+/* Day 3 Project */
+SELECT name FROM animals A 
+JOIN owners O ON O.id = A.owner_id AND O.full_name = 'Melody Pond';
+
+SELECT * FROM animals A
+JOIN species S ON A.species_id = S.id AND A.species_id = 1;
+
+SELECT owners.full_name, animals.name FROM animals
+FULL JOIN owners ON owners.id = animals.owner_id;
+
+SELECT COUNT(species_id) FROM animals
+FULL JOIN species ON species.id = animals.species_id GROUP BY species_id;
+
+SELECT name FROM animals
+RIGHT JOIN owners ON owners.id = animals.owner_id AND owner_id = 2 AND species_id = 2;
+
+SELECT name FROM animals
+JOIN owners ON owners.id = animals.owner_id AND escape_attempts = 0 AND owner_id = 5;
+
```

Comment:
- [ ] Instead of adding a condition clause for the `owner_id`, kindly use the name of the owner instead of its `id`.